### PR TITLE
No "Ae" --> "AE" translate

### PR DESCRIPTION
**Pull request by tritoch**
Respect the 'Ae'

### CODE REVIEW

**Comment by Daenyth** [comment]
This should also fix #2161

File: common/decklist.cpp [hunk 1]
```diff
@@ -579,8 +579,6 @@ bool DeckList::loadFromStream_Plain(QTextStream &in)
         cardName.replace(rx, "'");
         rx.setPattern("Æ");
         cardName.replace(rx, "AE");
```
Comment (Psithief) [inline common/decklist.cpp]: Shouldn't this be `Ae` as well?

Comment (Daenyth) [inline common/decklist.cpp]: I'd prefer to leave that until we know it's an issue - this should only affect much older cards

Comment (Psithief) [inline common/decklist.cpp]: I suppose we need to know what the data source (mtgjson) plans to do about this first.

Comment (tooomm) [inline common/decklist.cpp]: I thought there are supposed to be no more `Æ` cards at all starting with Kaladesh?
(besides the ordinary errors on gatherer for sure...)

**Comment by tritoch** [comment]
>no more AE cards on gatherer

That's correct (or should/will be) but this replacement is for deck editor.  It needs to handle people who made deck lists before gatherer was fixed or from other programs which may not have been updated.  The other replace should be 'Ae' I will update that asap.

**Comment by tooomm** [comment]
Can you give a bit more details what your change is about in the PR next time, please?

Whats the problem, what does it fix, why is it needed... just like this: :P
>(...) **but this replacement is for deck editor. It needs to handle people who made deck lists before gatherer was fixed or from other programs which may not have been updated.**

**Comment by ZeldaZach** [comment]
https://www.youtube.com/watch?v=1umfD-k2lww&feature=youtu.be was fixed with this PR, i assume?

**Pull request merged**